services/elm-package/elm-package.service.js

```diff
@@ -12,8 +12,8 @@ module.exports = class ElmPackage extends LegacyService {
       cache((data, match, sendBadge, request) => {
         const urlPrefix = 'http://package.elm-lang.org/packages'
```

Comment:
Is it possible to use the https version here?

Reply:
Yes, if that is supported, that would be great!

Reply:
The https URL is online, but I don't know if shields handles it. I guess I'll just change this, since I have to trigger the tests anyway.